desktop/src/main/java/bisq/desktop/main/portfolio/pendingtrades/TradeSubView.java

```diff
@@ -108,7 +108,6 @@ private void buildViews() {
         SimpleMarkdownLabel label = addSimpleMarkdownLabel(leftGridPane, ++leftGridPaneRowIndex);
         AutoTooltipButton button = (AutoTooltipButton) addButtonAfterGroup(leftGridPane, ++leftGridPaneRowIndex, "");
         SimpleMarkdownLabel footerLabel = addSimpleMarkdownLabel(leftGridPane, ++leftGridPaneRowIndex, Res.get("portfolio.pending.stillNotResolved"), 10);
-        footerLabel.getStyleClass().add("small-text");
```

Comment:
I'm afraid there is not enough space for making this text bigger. This change cause that there is no space between `Need help?` heading and paragraph (https://github.com/bisq-network/bisq/pull/5985#discussion_r796824906)

Reply:
I think using converted 11px instead of 10px is fine for the sub text IMO.

![Bildschirmfoto 2022-02-09 um 10 20 22](https://user-images.githubusercontent.com/170962/153164546-58acd322-9064-4a36-ac31-1da1d5ee07a9.png)
![Bildschirmfoto 2022-02-09 um 10 20 27](https://user-images.githubusercontent.com/170962/153164554-0a039f50-5c42-436b-bc3b-3c0e7745ffef.png)

But it is still not enough for all use cases as you can see in the screenshots. So I'd just increase the SubView min height in that case.
![Bildschirmfoto 2022-02-09 um 10 44 40](https://user-images.githubusercontent.com/170962/153170432-0c275d9e-bddd-49f3-87ad-3ff61a4124ab.png)
![Bildschirmfoto 2022-02-09 um 10 44 37](https://user-images.githubusercontent.com/170962/153170447-ecae23ee-b71e-462e-b706-5a64fb9cfabd.png)